src/api/app/policies/project_policy.rb

```diff
@@ -35,6 +35,10 @@ def unlock?
     user.can_modify?(record, true)
   end
 
+  def source_access?
+    !(!User.current.is_admin? && record.disabled_for?('sourceaccess', nil, nil))
```

Comment:
f*** - you really try my limits here, no? :)

Because to me this is a pretty complicated way to write
`User.current.is_admin? || !record.disabled_for?('sourceaccess', nil, nil)`

Reply:
I'm not sure what you are talking about (now making sure that I saved the changes that I did with @DavidKang, one hour ago) :)